### PR TITLE
feat(init): initialize theme

### PR DESCRIPTION
**Pull request by benjamingarcia**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,9 +12,9 @@ All instructions can be found at [draculatheme.com/nylas-n1](https://draculathem
 
 This theme is maintained by the following person(s) and a bunch of [awesome contributors](https://github.com/dracula/template/graphs/contributors).
 
-[![Harrison Heck](https://avatars0.githubusercontent.com/u/1037526?v=3&s=70)](https://github.com/nesl247) | [![Zeno Rocha](https://avatars2.githubusercontent.com/u/398893?v=3&s=70)](https://github.com/zenorocha)
+[![Harrison Heck](https://avatars0.githubusercontent.com/u/1037526?v=3&s=70)](https://github.com/nesl247) | [![Zeno Rocha](https://avatars2.githubusercontent.com/u/398893?v=3&s=70)](https://github.com/zenorocha) | [![Benjamin Garcia](https://avatars3.githubusercontent.com/u/1433960?v=3&amp;s=460)](https://github.com/benjamingarcia/)
```
Comment (nesl247) [inline README.md]: Please remove Zeno and I as maintainers here. It should only be you listed.

File: ui-variables.less [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+  background-color: ghostwhite;
+}
+
+p.MsoNormal, div.participants.message-participants {
+  color: @background-secondary !important;
+}
+
+span.participant-name.from-contact {
+  color: @background-secondary !important;
+}
```
Comment (nesl247) [inline ui-variables.less]: Should have an ending newline no?

Comment (benjamingarcia) [inline ui-variables.less]: done! 👍

File: README.md (hunk 1 above)
Comment (benjamingarcia) [inline README.md]: all right :metal:

File: README.md [hunk 3]
```diff
@@ -12,9 +12,9 @@ All instructions can be found at [draculatheme.com/nylas-n1](https://draculathem
 
 This theme is maintained by the following person(s) and a bunch of [awesome contributors](https://github.com/dracula/template/graphs/contributors).
 
-[![Harrison Heck](https://avatars0.githubusercontent.com/u/1037526?v=3&s=70)](https://github.com/nesl247) | [![Zeno Rocha](https://avatars2.githubusercontent.com/u/398893?v=3&s=70)](https://github.com/zenorocha)
---- | ---
-[Harrison Heck](https://github.com/nesl247) | [Zeno Rocha](https://github.com/zenorocha)
+[![Benjamin Garcia](https://avatars3.githubusercontent.com/u/1433960?v=3&amp;s=460)](https://github.com/benjamingarcia/)
+---
```
Comment (nesl247) [inline README.md]: Missing the `|` at the end here

File: README.md [hunk 4]
```diff
@@ -12,9 +12,9 @@ All instructions can be found at [draculatheme.com/nylas-n1](https://draculathem
 
 This theme is maintained by the following person(s) and a bunch of [awesome contributors](https://github.com/dracula/template/graphs/contributors).
 
-[![Harrison Heck](https://avatars0.githubusercontent.com/u/1037526?v=3&s=70)](https://github.com/nesl247) | [![Zeno Rocha](https://avatars2.githubusercontent.com/u/398893?v=3&s=70)](https://github.com/zenorocha)
---- | ---
-[Harrison Heck](https://github.com/nesl247) | [Zeno Rocha](https://github.com/zenorocha)
+[![Benjamin Garcia](https://avatars3.githubusercontent.com/u/1433960?v=3&amp;s=460)](https://github.com/benjamingarcia/)
+--- 
+[Benjamin Garcia](https://github.com/benjamingarcia)
```
Comment (nesl247) [inline README.md]: Missing the `|` at the end here. These are causing the table to not display correctly.

**Comment by benjamingarcia** [comment]
I fix array.

**Pull request merged**